app/services/symbolService.js

```diff
@@ -131,51 +132,52 @@ class SymbolService {
     
     let usdtRate = 1; 
     if (quote !== 'USDT') {
-      const ticker = await exchange.fetchTicker(symbol + '/USDT');
+      const ticker = await exchange.fetchTicker(`${symbol  }/USDT`);
```

Comment:
this alternation should be fine, but you need to it should be this ${ symbol }/USDT`).

Reply:
> this alternation should be fine, but you need to it should be this ${ symbol }/USDT`).

done